### ISSUE TITLE
Feature Request: Add split window functionality

### ISSUE THREAD

**Issue by brandor5** [lanoxx/tilda#199]
Would it be possible to add the ability to split a window horizontally and vertically?

Thanks!

**Comment by lanoxx** [comment]
There are already shell based solutions for this, like using tmux, have you tried them?

**Comment by brandor5** [comment]
Yes, I do use tmux when I'm doing remote work and splitting is ok for that.

I'm really looking for something that I can use locally on my laptop that mimics yakuake.

**Comment by lanoxx** [comment]
I just tried that and actually that is a very nice feature. I am sure it could be implemented in tilda too, but not sure how much work it would be. If someone wants to implement this and send a pull request feel free to do so. Otherwise I might implement it when I have time. No promise though.

**Comment by jsayol** [comment]
You can just use tmux inside Tilda. I just tried and it works perfectly fine. I think it'd be better to use existing tools rather than implement this functionality in Tilda.

![Screenshot](https://i.imgur.com/EGl7m5x.png)
